reactions.json

```diff
@@ -208,6 +208,34 @@
     "KI": "iodobenzene",
     "HBF4 / (Heat)": "fluorobenzene"
   },
+  "but-2-ene": {
+    "H2O / dil H2SO4": "butan-2-ol",
+    "PCC": "butanone"
+  },
+  "acetone": {
+    "H2O / LiAlH4": "propan-2-ol",
+    "conc H2SO4 (443K)": "propene",
+    "O3 / Zn dust / H2O": "ethanal",
+    "H+ / KMnO4": "ethanoic acid"
+  },
+  "propan-1-ol": {
+    "PCl5": "1-chloropropane",
+    "Mg / Dry ether": "propylmagnesium chloride",
+    "formaldehyde / H2O": "butan-1-ol",
+    "H+ / KMnO4": "butanoic acid"
+  },
+  "ethanal": {
+    "NaOH": "3-hydroxybutanal",
+    "I2 (343K)": "but-2-enal",
+    "Br2 / CCl4": "2,3-dibromobutanal",
+    "alc. KOH": "but-2-ynal"
+  },
+  "benzoic acid": {
+    "LiAlH4": "phenylmethanol",
+    "PCl5": "benzyl chloride",
+    "Mg / Dry ether": "benzylmagnesium chloride",

```

Comment:
How did it form a chloride here? neither the substrate nor the reagent has chlorine.

Reply:
LiAlH4 first refuses the carboxylic group to alcohol, then chloride forms due to the PCl5 reagent

Reply:
I think you made a mistake in understanding the json structure. Each key and value denotes a seperate reaction and its not a chain reaction as you've put it. 
What you should've written is 
```json
  "benzoic acid": {
    "LiAlH4": "phenylmethanol"
   },
  "phenylmethanol": {
      "PCl5": "benzyl chloride"
  },
  "benzyl chloride": {
      "Mg / Dry ether": "benzylmagnesium chloride"
  },
```

You'll need to edit all the 5 reactions in this format.

Reply:
I am so sorry. All 5 contributions are misleading. I'll fix them all again